source/uwp/Renderer/idl/AdaptiveCards.XamlCardRenderer.idl

```diff
@@ -777,6 +793,7 @@ namespace AdaptiveCards
         interface IAdaptiveActionEventArgs : IInspectable
         {
             [propget] HRESULT Action([out, retval] IAdaptiveActionElement** action);
+            [propget] HRESULT Inputs([out, retval] HSTRING* value);
```

Comment:
Hmm, for UWP that makes it much more difficult for apps to process the inputs. Action.Submit would be used for foreground and background activations, where apps need to pull out what the user typed from a text box and send it as a message, for example. Putting it in JSON instead of a KeyValuePair makes it much more difficult to process.

If you want to check this in now you can, but I think we'll need to change it in the future. Hopefully @matthidinger can chime in.